test/program_test.exs

```diff
@@ -14,4 +14,13 @@ defmodule WaspVM.ProgramTest do
     assert result_2 == 2
   end
 
+  test "Trace Works" do
+    {:ok, pid} = WaspVM.start()
+    WaspVM.load_file(pid, "test/fixtures/wasm/int_div.wasm")
+    {status, gas, result} = WaspVM.execute(pid, "main", [-4], [trace: true])
+
+
+    assert result == -2
```

Comment:
We should actually read the trace file in and make sure things got written to it correctly (and don't forget to have the file deleted automatically after the test finishes running)

Reply:
Appended test to read the outputted file, check for > 0 string size, and delete test file on close